gateway/gateway.go

```diff
@@ -151,7 +151,7 @@ func (gateway *HandleT) initUserWebRequestWorkers() {
 		logger.Debug("User Web Request Worker Started", i)
 		var userWebRequestWorker *userWebRequestWorkerT
 		userWebRequestWorker = &userWebRequestWorkerT{
-			webRequestQ:    make(chan *webRequestT),
+			webRequestQ:    make(chan *webRequestT, maxUserWebRequestBatchSize),
```

Comment:
Let's have a comment on how we came up with this batch sizes.

Reply:
Also write a comment on why it could be a good idea to make done a buffered channel of size 1.